doc/userdoc/styleguide/styleguide.rst

```diff
@@ -0,0 +1,566 @@
+
+
+For example:
+
+.. code-block:: none
+
+    The following example is based on Smith [1]_.
+    [2]_ contains a technically detailed description.
+
+Please ensure your reference follows the following guidelines:
+
+* References with more than 5 authors use 'et al.'.
+* Use initials for first name of authors
+* Surname precedes first name for all authors
+* No comma follows surname
+* Full stop after every section of bibliography.
+* No formatting such as italics, bold, or underline.
+* Full title of journal
+* Article titles written in sentence case
+* Year follows author(s), in parentheses
+* Volume, can be optionally followed by issue in parentheses,  a colon
+  separates volume and page range.
+* Include a linked DOI, if available
+
+.. code-block:: none
+
+ References
+ -----------
+
+ .. [1] Smith J. and Jones M (2009). Title of cool paper. Journal of
+        Awesomeness. 3:7-29. <DOI>
+
+ .. [2] Sander M., et al (2011). Biology of the sauropod dinosaurs: the
+        evolution of gigantism. Biological Reviews. 86(1):117-155.
+        https://doi.org/10.1111/j.1469-185X.2010.00137.x
+
+How we write
+------------
+
+Headings and subheadings
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Headings and subheadings should describe the purpose of the section.
+
+Begin with a descriptive verb or begin with `How to ...`
+
+Headings should explain the section in a short phrase.
+
+Use the verb stem and not the gerund ('ing') form of verbs. Not 'Adding a
+model', but  'Add a model'.
+
+Avoid section names like `Introduction` or `Part 1`.
+
+One word subheadings are acceptable, if the section is short and the meaning is clear.
+
+Use sentence case for headings and subheadings, i.e., begin with an uppercase
+letter but with all other words in lower case (except proper nouns).
+
+
++-----------------------------------+----------------+
+| Good examples:                    | Bad examples:  |
++===================================+================+
+| Create your first neural network  | Start here     |
++-----------------------------------+----------------+
+| How to set up and configure MUSIC | MUSIC and NEST |
++-----------------------------------+----------------+
+| Add a device to your network      | Adding devices |
++-----------------------------------+----------------+
+
+
+Numbers
+~~~~~~~
+
+Numbers should be spelled out if they begin a sentence. In most cases, however,
+the numeral/ordinal format is preferred.
+
+For additional mathematical notation, use the math role or directive.
+
+We use the period for the decimal point. (`57.45`)
+
+The thousand separator is the comma except when showing a code example
+
+   Example:
+
+   We have over 5,000 connections.
+   The number of connections is ``x = 5000``
+
+Make sure you use the correct unit (e.g., millivolts for voltage) and the
+unit's syntax (`V_m`).
+
+Lists
+~~~~~
+
+Use ordered lists for step-by-step instructions only. Do not have more that 2
```

Comment:
ordered = numbered lists?

Reply:
yes, numbered is much more clear than ordered